channel-adapter/channel-adapter.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/io;
+import ballerinax/jira;
+
+configurable string username = "admin@com.baclmock.io";
+configurable string password = "admin123";
```

Comment:
"admin123" still looks like it could be a user-provided password, not recommended to put in the source. please use  "admin".

let's make the user name also "admin" to make it obvious. real password should come form cli.